.github/workflows/continuous-integration.yml

```
@@ -18,6 +17,18 @@ jobs:
         with:
           args: --from=${{ github.event.pull_request.base.sha }}
 
+  composer_normalize:
+    name: "Composer Normalize"
+    runs-on: "ubuntu-latest"
```

We usually pin those:

```suggestion
    runs-on: "ubuntu-20.04"
```